examples/client/Locomotion/src/pages/ActiveRide/newMap.js

```diff
@@ -165,7 +176,7 @@ export default React.forwardRef(({
         customMapStyle={isDarkMode ? mapDarkMode : undefined}
         {...mapSettings}
       >
-        {requestStopPoints.filter(sp => !!sp.lat).length > 1
+        {currentBsPage !== BS_PAGES.CONFIRM_PICKUP && requestStopPoints.filter(sp => !!sp.lat).length > 1
```

Comment:
extract to a variable and use it.
isConfirmPickup = currentBsPage === BS_PAGES.CONFIRM_PICKUP